packages/ses/src/error/fatal-assert.js

```diff
@@ -10,7 +10,7 @@ let abandon;
 // below). Currently it only checks for the `process.abort` or `process.exit`
 // found on Node. It should also sniff for a vat terminating function expected
 // to be found within the start compartment of SwingSet vats. What else?
-if (typeof process === 'object') {
+if (process && typeof process === 'object') {
```

Comment:
This is a behavioral change that will cause a `ReferenceError` if not defined.

Reply:
Ack, nice catch!